Client/Frontend/Home/BookmarksPanel.swift

```diff
@@ -97,6 +97,8 @@ class BookmarksPanel: SiteTableViewController, HomePanel {
             if let bookmark = source.current[indexPath.row] {
                 if let favicon = bookmark.favicon {
                     cell.imageView?.setIcon(favicon, withPlaceholder: self.defaultIcon)
+                } else {
+                    cell.imageView?.image = defaultIcon
```

Comment:
This whole clause from line 98 on turns into:

```
    cell.imageView.setIcon(bookmark.favicon, withPlaceholder: self.defaultIcon)
```

Read the definition of `setIcon`.

Try that. If it works, land it.